iis-workflows/iis-workflows-transformers/src/test/java/eu/dnetlib/iis/workflows/transformers/spark/citationmatching/direct/SparkCitationMatchingDirectTransformerWorkflowTest.java

```diff
@@ -0,0 +1,22 @@
+package eu.dnetlib.iis.workflows.transformers.spark.citationmatching.direct;
+
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import eu.dnetlib.iis.IntegrationTest;
+import eu.dnetlib.iis.core.AbstractOozieWorkflowTestCase;
+
+/**
+ * 
+ * @author madryk
+ *
+ */
+@Category(IntegrationTest.class)
+public class SparkCitationMatchingDirectTransformerWorkflowTest extends AbstractOozieWorkflowTestCase {
+
+	@Test
+	public void testTransformer() {
+		
+		testWorkflow("eu/dnetlib/iis/workflows/transformers/spark/citationmatching/direct/with_sample_data");
```

Comment:
I wonder if the subfolder "with_sample_data" is necessary. Couldn't the test app be placed just under direct?